src/AuditReader.php

```diff
@@ -708,7 +690,7 @@ public function getEntityValues($className, $entity)
      * @phpstan-param class-string<T>              $className
      * @phpstan-return array<T|null>
      */
-    public function getEntityHistory($className, $id)
+    public function getEntityHistory($className, int|string|array $id)
```

Comment:
I think a lot of this changes are not safe to do in this branch.

Reply:
We have phpstan at a good level, a 86% coverage.
And I kinda consider our BC policy should only be for people using the code according to the phpdoc.

Reply:
It is not final and we never did a major of this bundle. Not sure if there is a way to avoid disable this rule.

Reply:
I've opened https://github.com/rectorphp/rector/issues/7823 but yes we can disable the rule anyway.